commit 080410fe61e6df035960f5cbec9e381ac8b4ced0
Author: Thorsten Blum <thorsten.blum@linux.dev>
Date:   Mon Apr 7 11:08:29 2025 +0200

    ALSA: azt2320: Replace deprecated strcpy() with strscpy()
    
    strcpy() is deprecated, use strscpy() instead.
    
    Link: https://github.com/KSPP/linux/issues/88
    Cc: linux-hardening@vger.kernel.org
    Signed-off-by: Thorsten Blum <thorsten.blum@linux.dev>
    Link: https://patch.msgid.link/20250407090832.743255-1-thorsten.blum@linux.dev
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

diff --git a/sound/isa/azt2320.c b/sound/isa/azt2320.c
index b937c9138d12..588b9f0831d3 100644
--- a/sound/isa/azt2320.c
+++ b/sound/isa/azt2320.c
@@ -189,8 +189,8 @@ static int snd_card_azt2320_probe(int dev,
 	if (error < 0)
 		return error;
 
-	strcpy(card->driver, "AZT2320");
-	strcpy(card->shortname, "Aztech AZT2320");
+	strscpy(card->driver, "AZT2320");
+	strscpy(card->shortname, "Aztech AZT2320");
 	sprintf(card->longname, "%s, WSS at 0x%lx, irq %i, dma %i&%i",
 		card->shortname, chip->port, irq[dev], dma1[dev], dma2[dev]);